controller/login.go

```diff
@@ -92,46 +92,7 @@ func redirectWithParams(ctx redirectContext, config auth.AuthServiceConfiguratio
 
 // Generate generates access tokens in Dev Mode
 func (c *LoginController) Generate(ctx *app.GenerateLoginContext) error {
-	var tokens app.AuthTokenCollection
-
-	testuser, err := generateUserToken(ctx, c.configuration, c.configuration.GetKeycloakTestUserName())
-	if err != nil {
-		log.Error(ctx, map[string]interface{}{
-			"err":      err,
-			"username": c.configuration.GetKeycloakTestUserName(),
-		}, "unable to get Generate User token")
-		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, errs.Wrap(err, "unable to generate test token ")))
-	}
-	// Creates the testuser user and identity if they don't yet exist
-	_, _, err = c.auth.CreateOrUpdateKeycloakUser(*testuser.Token.AccessToken, ctx)
-	if err != nil {
-		log.Warn(ctx, map[string]interface{}{
-			"err":      err,
-			"username": c.configuration.GetKeycloakTestUserName(),
-		}, "unable to create or update user")
-	}
-	tokens = append(tokens, testuser)
-
-	testuser, err = generateUserToken(ctx, c.configuration, c.configuration.GetKeycloakTestUser2Name())
-	if err != nil {
-		log.Error(ctx, map[string]interface{}{
-			"err":      err,
-			"username": c.configuration.GetKeycloakTestUser2Name(),
-		}, "unable to generate test token")
-		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, errs.Wrap(err, "unable to generate test token")))
-	}
-	// Creates the testuser2 user and identity if they don't yet exist
-	_, _, err = c.auth.CreateOrUpdateKeycloakUser(*testuser.Token.AccessToken, ctx)
-	if err != nil {
-		log.Warn(ctx, map[string]interface{}{
-			"err":      err,
-			"username": c.configuration.GetKeycloakTestUser2Name(),
-		}, "unable to create or update user")
-	}
-	tokens = append(tokens, testuser)
-
-	ctx.ResponseData.Header().Set("Cache-Control", "no-cache")
-	return ctx.OK(tokens)
+	return proxy.RouteHTTPToPath(ctx, c.configuration.GetAuthServiceURL(), authservice.GenerateTokenPath())
 }
 
 func generateUserToken(ctx context.Context, configuration loginConfiguration, username string) (*app.AuthToken, error) {
```

Comment:
I guess we can remove this now? Or it's still used somewhere?

Reply:
We could probably remove the whole assets stuff as well. That little http page with some js stuff to show the generated tokens. I don't think this was ever used. But @ldimaggi should know what is used in e.g. tests

Reply:
tracking it here https://github.com/fabric8-services/fabric8-wit/issues/1759